src/pretix/base/models/event.py

```diff
@@ -453,6 +453,8 @@ def set_defaults(self):
         self.settings.ticketoutput_pdf__enabled = True
         self.settings.ticketoutput_passbook__enabled = True
         self.settings.event_list_type = 'calendar'
+        self.settings.invoice_email_attachment = True
+        self.settings.name_scheme = 'given_family'
```

Comment:
I think this should be added in a follow up PR.

Reply:
Hi @mariobehling , did not get your point, name_schema 'given_family' is default for new event, what need to be change?